typescript/sdk/src/utils/ism.ts

```diff
@@ -11,3 +13,61 @@ export function multisigIsmVerificationCost(m: number, n: number): number {
   // @ts-ignore
   return multisigIsmVerifyCosts[`${n}`][`${m}`];
 }
+
+// Function to recursively remove 'address' properties and lowercase string properties
+export function normalizeConfig(obj: WithAddress<any>): any {
+  return sortValidatorsAndModulesInConfig(lowerCaseConfig(obj));
+}
+
+function lowerCaseConfig(obj: any): any {
+  if (Array.isArray(obj)) {
+    return obj.map(normalizeConfig);
+  } else if (obj !== null && typeof obj === 'object') {
+    const newObj: any = {};
+    for (const key in obj) {
+      if (key !== 'address') {
+        newObj[key] = key === 'type' ? obj[key] : normalizeConfig(obj[key]);
+      }
+    }
+    return newObj;
+  } else if (typeof obj === 'string') {
+    return obj.toLowerCase();
+  }
+
+  return obj;
+}
+
+// write a function that will go through an object and sort any arrays it finds
+export function sortValidatorsAndModulesInConfig(config: any): any {
```

Comment:
nit maybe something like sortArraysInConfig, in case have to add more cases in the future
```suggestion
export function sortValidatorsAndModulesInConfig(config: any): any {
```